Resources/config/services.xml

```diff
@@ -50,6 +50,9 @@
         <service id="jms_translation.loader.xliff_loader" class="%jms_translation.loader.xliff_loader.class%" public="false">
             <tag name="jms_translation.loader" format="xliff" />
         </service>
+        <service id="jms_translation.loader.xliff_loader" class="%jms_translation.loader.xliff_loader.class%" public="false">
+            <tag name="jms_translation.loader" format="xlf" />
+        </service>
         
```

Comment:
doesn't this simply override the previous service with the same id and thus change the format to XLF from XLIFF?